src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+                    parsedDescription += " (by: " + dueDate + ")";
+                    // Remove the word 'deadline' from the description
+                    parsedDescription = parsedDescription.substring(parsedDescription.indexOf(" ") + 1);
+                    System.out.println(BARRIER + "\n");
+                    Deadline temp = new Deadline(parsedDescription, dueDate);
+                    System.out.println("added: " + temp.printTask());
+                    printListLength();
+                    System.out.println(BARRIER + "\n");
+                    break;
+                }
+
+                // Adds a new Event task
+                case "event": {
+                    // Split input into description, start, and end
+                    String[] splitEventInput = originalInput.split("/");
+                    // Start time comes after '/from'
+                    String start = splitEventInput[1].substring(5);
+                    // End time comes after '/to'
+                    String end = splitEventInput[2].substring(3);
+                    String parsedDescription = splitEventInput[0] + "(from: " + start + "to: " + end + ")";
+                    parsedDescription = parsedDescription.substring(parsedDescription.indexOf(" ") + 1);
+                    System.out.println(BARRIER + "\n");
+                    Event temp = new Event(parsedDescription, start, end);
+                    System.out.println("added: " + temp.printTask());
+                    printListLength();
+                    System.out.println(BARRIER + "\n");
+                    break;
+                }
+
+                // Adds a new task if no case is hit
+                default: {
+                    System.out.println(BARRIER + "\n");
+                    Task temp = new Task(input);
+                    System.out.println("added: " + temp.printTask());
+                    printListLength();
+                    System.out.println(BARRIER + "\n");
+                }
+            }
+
+            // Reads next input
+            input = CONSOLE.nextLine();
+        }
+
+        System.out.println(EXIT);
     }
-}
+
+    // Local Duke method to list all functions
+    public static void printHelpList() {
+        String BARRIER = "____________________________________________________________";
+        System.out.println(BARRIER + "\n\nAvast! Here be the commands ye can use to make me do yer bidding!\n- list: lists all current tasks\n- mark x: marks task x as complete\n- unmark x: unmarks task x as complete\n- todo 'description': adds a task to do with the given description\n- deadline 'description' /by 'deadline': adds a deadline task with the given date and description\n- event 'description' /from 'start' /to 'end': adds an event with the start and endtime\n- bye: exits Duke\n- anything else: adds a basic task with the given description\n" + BARRIER + "\n");
```

Comment:
Line length should be no longer than 120 chars. Use line wrapping at appropriate place on the sentence